docs/developer-guide/etendo-classic/how-to-guides/how-to-create-a-pick-and-execute-process.md

```diff
@@ -0,0 +1,301 @@
+---
+tags: 
+  - PickandExecute
+  - Process definition
+  - Action handler
+  - Actions
+  - Standard UI pattern
+---
+#  How to Create a Pick and Execute Process
+
+  
+##  Overview
+
+**Pick and Execute (P&E)** is a case of  **Process Definition** with **Standard UI pattern**.
+
+This section will add a new Pick and Execute process and associate it with the **Sales Order window**.
+
+The implementation requires development experience. 
+
+The following concept pages provide background information on action handlers and javascript
+development:
+
+  * [Action Handler](../../how-to-guides/example-page.md)
+  * [Client Side Development and API](../../concepts/client-side-development-and-api.md)
+  * [JavaScript Coding Conventions](../../concepts/javascript-coding-conventions.md)
```

Comment:
```suggestion
  * [JavaScript Coding Conventions](../concepts/javascript-coding-conventions.md)
```

Reply:
@Tomas-Vesco If the name of the file is changed in the path, the name of the actual file and the mkdocs.yml file must also be changed, but it should not be uncommented from the latter file if it has not yet been revoked.